init/fluent_bit_init_entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+./init/fluent_bit_init_process
+cat /init/fluent-bit-init.conf
```

Comment:
Why cat the file in entrypoint? This means it will be printed to the users log... which isn't bad but its also not really needed... and the user can get Fluent Bit to output all registered plugins if htey run it with debug logging...

Reply:
This came from a discussion with Bingfeng, he asked me how customer can quickly know which config files they are using.. Maybe using `cat` here is not suitable. Will delete